scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -284,7 +396,7 @@ Function PrepareTestFromArtifacts
                 Write-Host "Copy deployment file from $QuickstartDeploymentArtifactFilePath"
                 Copy-Item $QuickstartDeploymentArtifactFilePath -Destination $DeploymentWorkingFilePath -Force
             }
-            "QuickstartCerts|TransparentGateway" # Only when $ProxyUri is specified
+            {$_ -in "QuickstartCerts","TransparentGateway"} # Only when $ProxyUri is specified
```

Comment:
why you need to change this line?

Reply:
The switch was originally using -wildcard and got switched to -regex in master, so the old syntax of using | does not work and I had to fix this.

Reply:
I am sure that this syntax "QuickstartCerts|TransparentGateway" should work with -regex option.  I tested before with below code block.

switch -regex ($testName) {
    "DirectMethod.*" { Write-Output "directmethod" }
    "QuickstartCerts|TransparentGateway" { Write-Output "quickstart certs or transparent gateway" }
}

When $testName = "QuickstartCerts" or "TransparentGateway", it should print out "quickstart certs or transparent gateway".  Please verify.